### PR TITLE
Fix Hawaii EITC refundability status

### LINKED ISSUE #3672: Fix Hawaii EITC refundability status
In the 0.631.0 version, the HI EITC should be non-refundable in 2021 and 2022.  The following note indicates that it was non-refundable before tax year 2023.

HI-EITC

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #3672

### CODE REVIEW

File: policyengine_us/parameters/gov/states/hi/tax/income/credits/refundable.yaml [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 values:
   2021-01-01:
+    - hi_food_excise_credit
+    - hi_cdcc
+    - hi_tax_credit_for_low_income_household_renters
+
+  2023-01-01:
     - hi_food_excise_credit
     - hi_cdcc
     - hi_eitc
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/credits/refundable.yaml:11]: Could you move this to the bottom?

File: policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+description: Hawaii provides the following non-refundable tax credits.
+values:
+  2021-01-01:
+    - hi_eitc
+  2023-01-01:
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml:5]: Do we need this? Do we have a basic integration test to check?
```suggestion
  2023-01-01: {}
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml (hunk 2 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml:5]: I dont think we need this, I believe Nikhil has added the feature of empty lists in yaml files, will add a test to double check

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/hi/tax/income/credits/non_refundable.yaml]: works but we need `[]`

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**